### PR TITLE
Update Malware Feed

### PR DESCRIPTION
**Pull request by Mawdac**
The main changes in this are:

- **fix: Better name conventions**
- **fix: Update malware mapping for better representation**
- **fix: Update ransomware mapping for better representation**

### CODE REVIEW

File: external-import/zerofox/src/collectors/mappers/botnetToInfrastructure.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def botnet_to_infrastructure(
     objects.append(botnet)
 
     ip_address = Infrastructure(
-        name=f"IP Address -- {entry.ip_address}",
+        name=f"{entry.ip_address}",
```
Comment (DNRRomero) [inline external-import/zerofox/src/collectors/mappers/botnetToInfrastructure.py:22]: would it be better an Infrastructure object with a `controls` relationship or an IP object with a `communicates-with` relationship?

File: external-import/zerofox/src/collectors/mappers/ransomwareToMalware.py [hunk 2]
```diff
@@ -11,8 +11,8 @@ def ransomware_to_malware(now: str, entry: Ransomware) -> List[stixMalware]:
         else ""
     )
     ransomware = stixMalware(
-        name=f"Ransomware {ransomware_name} -- {entry.sha256}",
-        description=f"Ransomware with note -- {entry.ransom_note}",
+        name=f"{ransomware_name}",
+        description=f"```{entry.ransom_note}```",
```
Comment (DNRRomero) [inline external-import/zerofox/src/collectors/mappers/ransomwareToMalware.py:15]: shouldn't this one include a similar knowledge graph (file + indicator + rels) as what you included in malware?

File: external-import/zerofox/src/collectors/mappers/malwareToMalware.py [hunk 3]
```diff
@@ -1,52 +1,63 @@
...
-                start_time=entry.created_at,
-            )
-            for domain_name in urls
-        ]
-        + malware_families
-        + [
+    file = File(
+        name=entry.sha256,
+        hashes={
+            "SHA-512": entry.sha512,
```
Comment (DNRRomero) [inline external-import/zerofox/src/collectors/mappers/malwareToMalware.py:23]: would there be a problem if any of these are empty?

**Review by DNRRomero — APPROVED** [review]
added some minor comments, very nice work!

**Review by henrycov — APPROVED** [review]

File: external-import/zerofox/src/collectors/mappers/malwareToMalware.py (hunk 3 above)
Comment (Mawdac) [reply external-import/zerofox/src/collectors/mappers/malwareToMalware.py]: Yep! Good call - updated this so the hashes (excluding sha256 are only included if they exist)

File: external-import/zerofox/src/collectors/mappers/ransomwareToMalware.py (hunk 2 above)
Comment (Mawdac) [reply external-import/zerofox/src/collectors/mappers/ransomwareToMalware.py:22]: Yes it should! Update the logic here

File: external-import/zerofox/src/collectors/mappers/botnetToInfrastructure.py (hunk 1 above)
Comment (Mawdac) [reply external-import/zerofox/src/collectors/mappers/botnetToInfrastructure.py:23]: Possibly - but I think that's something I'll leave out of the scope of this effort

**Review by DNRRomero — APPROVED** [review]

**Pull request merged**